backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5713345_AddMaturingTablesAndConfigs.sql

```diff
@@ -0,0 +1,3 @@
+-- Window: Reiferdisposition, Overrides_Window_ID=NULL
+UPDATE AD_Window SET Overrides_Window_ID=NULL WHERE AD_Window_ID=541756
+;
```

Comment:
Like this it will still fail on rollout, if an custom pp_order_candidate window is created in an script with lower number. You also need to remove it from existing insert.